chat-client-ui-types/src/uiContracts.ts

```diff
@@ -27,6 +27,7 @@ export const GENERIC_COMMAND = 'genericCommand'
 export const CHAT_OPTIONS = 'chatOptions'
 export const DISCLAIMER_ACKNOWLEDGED = 'disclaimerAcknowledged'
 export const CHAT_PROMPT_OPTION_ACKNOWLEDGED = 'chatPromptOptionAcknowledged'
+export const STOP_RESPONSE = 'stopResponse'
```

Comment:
does it make sense for this to be STOP_RESPONSE to be more generic? or should it be stopChatResponse which is closer to mynah UI's name for the event that will send it?

Reply:
Is there a use case for this outside of stopping a chat response? If not, I think mirroring mynah UI could be helpful?

Reply:
Not sure, I've just updated it to mirror mynah UI. I guess since this is the chat clients package it makes sense that it would stopChatResponse anyways